### ISSUE TITLE
Embed images as b64

### ISSUE THREAD

**Issue by adamcvolz** [ssine/pptx2md#72]
For many reasons, it would be a beneficial option to be able to embed images as base64 encoded strings in markdown/html format, rather than have to specify a folder location to download them.

**Comment by jacks-sam1010** [comment]
Hello, we tried to solve the issue.

This is what we did:

Implement the ability to embed images as base64 encoded strings in markdown/html format. This change involves modifying the ConversionConfig, parser, and outputter to support this new feature.

You can review changes in this commit: https://github.com/jacks-sam1010/ssine-pptx2md-72/commit/bde233e0b49a01e426dde531011bb508174fb9cd.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

Latta AI seeks to solve problems in open source projects as part of its mission to support developers around the world. Learn more about our mission at https://latta.ai/ourmission . If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by iamyx123** [comment]
Yes, this function is especially useful when uploading PPT to AI models
